src/dao/pda_getters/proxy_rewarder_Pda.ts

```diff
@@ -0,0 +1,28 @@
+import { PublicKey } from '@solana/web3.js';
+import * as Seeds from './seeds';
+import { STAR_ATLAS_ADDRESSES } from '../constants';
+
+/**
+ * Returns the public key for the proxy account
+ */
+export const findProxyAddress = async (
+  escrowKey: PublicKey,
+  user: PublicKey
+): Promise<[PublicKey, number]> => {
+  return await PublicKey.findProgramAddress(
```

Comment:
`await` is redundant if you are returning a promise. (There are other cases of this in the code, but just making a single comment for brevity. You can find them by searching `return await`)
```suggestion
  return PublicKey.findProgramAddress(
```